src/index.html

```diff
@@ -13,7 +13,12 @@
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-
+  <label class="big-lab" data-qa="big">
+    <input type="text" placeholder="Try &#34;Los Angeles&#34;" data-qa="keypress" class="search-bar big">
```

Comment:
it's will be better to read
```suggestion
    <input type="text" placeholder="Try &#34;Los Angeles&#34;" data-qa="keypress" class="search-bar__input search-bar__input_big">
```